docs/linting/Troubleshooting.mdx

```diff
@@ -249,3 +249,9 @@ Rules such as [`no-unsafe-argument`](https://typescript-eslint.io/rules/no-unsaf
 ## My linting feels really slow
 
 If you think you're having issues with performance, see our [Performance Troubleshooting documentation](./troubleshooting/Performance.md).
+
+## Are TypeScript project references supported?
+
+No, TypeScript project references is not yet supported. 
+
+See [this issue](https://github.com/typescript-eslint/typescript-eslint/issues/2094 "typescript-eslint issue 2094 (project references support)") for more details.
```

Comment:
See https://github.com/typescript-eslint/typescript-eslint/pull/6771#discussion_r1148422979

Reply:
Just to clarify, would it be something like this that'd be more helpful?
```suggestion
See [this issue regarding project references](https://github.com/typescript-eslint/typescript-eslint/issues/2094 "typescript-eslint issue 2094 (project references support)") for more details.
```

Reply:
It would, yes. @Josh-Cena is right that the title tag I suggested is unnecessary. How about...

Just to clarify, would it be something like this that'd be more helpful?
```suggestion
See [issue #2094 discussing project references](https://github.com/typescript-eslint/typescript-eslint/issues/2094) for more details.
```